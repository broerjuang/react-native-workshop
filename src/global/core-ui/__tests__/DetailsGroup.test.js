import React from 'react';
import renderer from 'react-test-renderer';
import {DetailsGroup} from '../index';
import {Text} from 'react-native';

describe('Icon', () => {
  it('should render Icon corectly', () => {
    let component = renderer.create(
      <DetailsGroup name={'test'} right={true}>
        <Text>asfsfsdf</Text>
      </DetailsGroup>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
