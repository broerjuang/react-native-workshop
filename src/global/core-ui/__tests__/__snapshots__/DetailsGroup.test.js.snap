// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render Icon corectly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "grey",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "padding": 15,
        "paddingTop": 30,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      test
    </Text>
    <View>
       
       
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "grey",
        "borderBottomWidth": 2,
        "flex": 1,
        "paddingBottom": 15,
        "paddingTop": 15,
      }
    }
  >
    <View
      style={
        Object {
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        asfsfsdf
      </Text>
    </View>
  </View>
</View>
`;
